ui/apps/platform/src/Containers/Violations/ViolationsTableSearchFilter.tsx

```diff
@@ -25,13 +25,19 @@ import {
     ID as DeploymentID,
     Name as DeploymentName,
 } from 'Components/CompoundSearchFilter/attributes/deployment';
+import { InactiveDeployment as AlertInactiveDeployment } from 'Components/CompoundSearchFilter/attributes/alert';
 
 const searchFilterConfig: CompoundSearchFilterConfig = [
     {
         displayName: 'Policy',
         searchCategory: 'ALERTS',
         attributes: [PolicyName, PolicyCategory],
     },
+    {
+        displayName: 'Policy Violation',
```

Comment:
We will display `Policy violation` because that's the terminology we use in the UI. Up to us to determine if we want to change `alert.ts` to `policyViolation.ts` and `AlertInactiveDeployment` to `PolicyViolationInactiveDeployment`. I'm ok with either approach. Both make sense to me.